src/shared/utils/dateTimeUtils.ts

```diff
@@ -16,3 +16,25 @@ export const timesNeedConverting = newTimeRange => {
   // they will be formatted the same
   return !isUtcTime(newTimeRange.lower)
 }
+
+// takes in Date object and adds a duration to it.
+// To add 1 hour, value = 1, unit = 'h'
+// similarly, to add a minute. value = 1, unit = 'm'
+export function incrementDate(input: Date, value: number, unit?: string): Date {
```

Comment:
When reading this method name, `incrementDate` sounds like i'm doing something like
```ts
const date = new Date()
date++
```

what do you think about something like

```ts
addDurationToDate(date: Date, duration: number, unit?: string): Date {
  const clonedDate = new Date(input)
  ...
```
? Is that the intent: we want to add a duration to a date (I personally prefer `duration` since it's less ambiguous than `value`). Your intent with `result = new Date(input)` is to clone the date so you don't change the argument passed in, right? I'd say use a variable name to make that clear.

Reply:
Yes, that method name and the signature makes more sense! We are adding a time duration to our date object.